Remotion/Web/Development.WebTesting/Utilities/ProcessUtils.cs

```diff
@@ -239,9 +239,9 @@ private static IReadOnlyList<Process> WaitForProcessesExits (IEnumerable<Process
     private static Exception CreateProcessesDidNotExitInTimeException (IEnumerable<Process> processes, TimeSpan timeout)
     {
       var stringBuilder = new StringBuilder();
-      stringBuilder.AppendLine ($"The following processes did not exit within the timeout of {timeout.TotalSeconds:0.##} seconds:");
+      stringBuilder.AppendFormat ("The following processes did not exit within the timeout of {0:0.##} seconds:", timeout.TotalSeconds);
       foreach (var process in processes)
-        stringBuilder.AppendLine ($" - Process '{process.ProcessName}' (id: {process.Id})");
+        stringBuilder.AppendFormat (" - Process '{0}' (id: {1})", process.ProcessName, process.Id);
```

Comment:
The `.AppendLine()` is missing at the end. That's the downside of using AppEndFormat, but downside that's acceptable.